hack/generate-antrea-samples.py

```diff
@@ -54,6 +54,6 @@ def str_presenter(dumper, data):
             out['spec']['antreaCNIConfig'] = doc.get('data', {}).get('antrea-cni.conflist', "")
             out['spec']['antreaControllerConfig'] = doc.get('data', {}).get('antrea-controller.conf', "")
 
-print(yaml.dump(out))
+print(yaml.dump(out, default_flow_style=False, allow_unicode=True))
```

Comment:
Is it worthwhile to apply formatting here as well?
https://github.com/vmware/antrea-operator-for-kubernetes/blob/main/hack/generate-role-yaml.py#L42

Reply:
I think we can keep it as there is no long string data in role.yaml.